src/Files.App/Helpers/ExternalResourcesHelper.cs

```diff
@@ -108,6 +109,22 @@ public async Task<bool> TryLoadThemeAsync(AppTheme theme)
 			}
 		}
 
+		/// <summary>
```

Comment:
Isn't there already a method for this? Can we reuse it?

Reply:
With this there are 3 similar methods, but they're all a bit different and I didn't want to revalidate them 😅